pkgs/development/mobile/androidenv/build-tools.nix

```diff
@@ -4,7 +4,10 @@ deployAndroidPackage {
   inherit package os;
   nativeBuildInputs = [ makeWrapper ]
     ++ lib.optionals (os == "linux") [ autoPatchelfHook ];
-  buildInputs = lib.optionals (os == "linux") [ pkgs.glibc pkgs.zlib pkgs.ncurses5 pkgsi686Linux.glibc pkgsi686Linux.zlib pkgsi686Linux.ncurses5 pkgs.libcxx ];
+  buildInputs = lib.optionals (os == "linux") (
+    (with pkgs; [ glibc zlib ncurses5 stdenv.cc.cc ])
+    ++ (with pkgsi686Linux; [ glibc zlib ncurses5 ])
+  );
```

Comment:
Likewise; this looks like it's just a formatting change (I think?).  Please put it in a separate PR.

Reply:
Those changes was in the spirit of Boy Scout Rule:

> The Boy Scouts have a rule: “Always leave the campground cleaner than you found it”

but anyway! I revert them back as you requested. check https://github.com/NixOS/nixpkgs/pull/226489/commits/398f142623dee16188587d66c83375c5fddb739c